test/run

```diff
@@ -22,6 +23,20 @@ import subprocess
 import sys
 import uuid
 
+DESCRIPTION = "Execute e2e and unit tests for Litani"
+
+
+def get_args():
+    pars = argparse.ArgumentParser(description=DESCRIPTION)
+    for arg in [{
+            "flags": ["--output-dir"],
+            "help": "output dir for test results",
+            "default": pathlib.Path(__file__).resolve().parent / "output"
```

Comment:
If the user supplies a value, it would be a `str`. But the default value you're giving here is a `pathlib.Path`. I don't like it when you're not sure what type an argument is, especially in cases like this: we can use the `/` operator on the output directory if it's a `Path` but not a `str`, for example.

Suggestions:
```
"default": str(pathlib.Path(__file__).resolve().parent / "output")
```
so you know that it will be a `str` either way, or
```
"type": pathlib.Path
```
so that you know it's a `Path` either way. (See [this](https://docs.python.org/3/library/argparse.html#type) for how that works)